packages/cli/src/cliEntry.js

```diff
@@ -140,8 +149,26 @@ async function run() {
     ? path.resolve(options.projectRoot)
     : process.cwd();
 
+  const reactNativePath = options.reactNativePath
+    ? path.resolve(options.reactNativePath)
+    : (() => {
+        try {
+          return path.dirname(
+            // $FlowIssue: Wrong `require.resolve` type definition
+            require.resolve('react-native/package.json', {
+              paths: [root],
+            })
+          );
+        } catch (_ignored) {
+          throw new Error(
+            'Unable to find React Native files. Have you installed project dependencies?'
```

Comment:
I am wondering what is the best description of this error. Usually it means there's no `react-native` in `node_modules` (e.g. they are empty, deleted or an error) or we are running in an environment where `react-native` cannot be found (it's not in the regular require.resolve hierarchy)